app/controllers/users/approvals_controller.rb

```diff
@@ -10,45 +10,36 @@ def new
   end
 
   def create
-    result = CreateApproval.call(
+    result = Users::Approvals::CreateUserApproval.call(
```

Comment:
CreateApproval was not originally in any modules, i found that once I moved it into Users/Approvals you have to include the namespace in order to call it